src/main/java/com/linecorp/armeria/client/RemoteInvokerOptions.java

```diff
@@ -66,6 +66,17 @@ public static RemoteInvokerOptions of(RemoteInvokerOptionValue<?>... options) {
         return new RemoteInvokerOptions(DEFAULT, options);
     }
 
+    /**
+     * Returns the {@link RemoteInvokerOptions} with the specified {@link RemoteInvokerOptionValue}s.
+     */
+    public static RemoteInvokerOptions of(Iterable<RemoteInvokerOptionValue<?>> options) {
+        if (options == null) {
```

Comment:
Is it deterministic which of this or the other of method null will be accepted by? Anyways, I think requireNonNull is fine, DEFAULT is public.